### PR TITLE
Reduce bundle size

### PR DESCRIPTION
**Pull request by uwolfer**

### CODE REVIEW

File: build/webpack.config.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ const webpackConfig = {
   resolve: {
     modules: [
       path.resolve(paths.client(), packageDir, 'src'),
-      'node_modules'
+      'node_modules',
```
Comment (MethodenMann) [inline build/webpack.config.js]: Is there a difference to:
path.resolve(paths.client(), packageDir, 'node_modules')

File: packages/entity-browser/package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "description": "Lists entities and allows to sort and search them",
   "main": "dist/index.js",
   "scripts": {
-    "clean": "rimraf dist",
+    "clean": "../../node_modules/rimraf/bin.js dist",
     "compile": "babel-node ../../bin/compile",
```
Comment (MethodenMann) [inline packages/entity-browser/package.json]: is this script necessary? if yes, where is the package name defined? compile script throws an error without a packagename provided

File: build/webpack.config.js (hunk 1 above)
Comment (uwolfer) [inline build/webpack.config.js]: I assume it's the same.

File: packages/entity-browser/package.json (hunk 2 above)
Comment (uwolfer) [inline packages/entity-browser/package.json]: Still required.

**Pull request merged**